db/migrate/20160412063136_add_user_relationships.rb

```diff
@@ -0,0 +1,10 @@
+class AddUserRelationships < ActiveRecord::Migration
+  def change
+    create_table :user_relationships do |t|
+      t.belongs_to :follower, index: true
+      t.belongs_to :following, index: true
```

Comment:
Should these be not null? I believe so.

Should there also not be unique index on the pair of these?